install_ddev.sh

```diff
@@ -46,46 +46,50 @@ ${GREEN}
 ${RESET}"
 while true; do
     read -p "Continue? (y/n): " INSTALL
-    case $INSTALL in
+    case "$INSTALL" in
         [Yy]* ) break;;
         [Nn]* ) exit;;
         * ) echo "Please answer y or n.";;
     esac
 done
 
-if ! docker run -t busybox:latest ls >/dev/null; then
-    printf "
-    ${RED}
-    ####
-    # You need to install Docker and have it running before executing this script.
-    # The installer may be provided with this package.
-    # Docker installation and troubleshooting information is at
-    # https://ddev.readthedocs.io/en/latest/users/docker_installation/
-    ####
-    ${RESET}"
-    exit 1
-fi
-
 echo ""
 echo "Installing docker images for ddev to use..."
-if [[ "$OS" == "Darwin" ]]; then
-    gzip -dc $(ls ddev_tarballs/ddev_docker_images*.tar.xz) | docker load
-elif [[ "$OS" == "Linux" ]]; then
-    xzcat $(ls ddev_tarballs/ddev_docker_images*.tar.xz) | docker load
-fi
+7z x ddev_tarballs/ddev_docker_images.*.tar.xz -so | docker load
 
-TARBALL="$(ls ddev_tarballs/$FILEBASE*.tar.gz)"
+TARBALL=""
+case "$OS" in
+    Linux)
+        TARBALL=ddev_tarballs/ddev_linux*.tar.gz
+        ;;
+    Darwin)
+        TARBALL=ddev_tarballs/ddev_macos*.tar.gz
+        ;;
+    MINGW64_NT-10.0)
+        echo "${YELLOW}PLease use the ddev_windows_installer provided with this package to install ddev${RESET}"
```

Comment:
The prompt on line 39 probably needs to be updated to mention that Windows users should install ddev. At the moment it mentions that the script installs ddev for them.